cartridges/int_adyen_overlay/cartridge/scripts/util/AdyenHelper.ds

```diff
@@ -446,6 +454,24 @@ var __AdyenHelper: Object = {
         return {error: true};
     },
 
+    add3DS2Data: function(jsonObject){
+        jsonObject["additionalData"].allow3DS2 = true;
+        jsonObject["channel"] = "web";
+
+        var card;
+        if (session.forms.billing.paymentMethods) {
+            card = session.forms.billing.paymentMethods.creditCard;
+        }
+        else {
+            card = session.forms.billing.creditCardFields;
+        }
+
+        jsonObject["browserInfo"] = card.browserInfo.value;
```

Comment:
I believe some of the browserInfo data needs to be retrieved from the backend. @cyattilakiss do we have docs on this part ?

Reply:
https://github.com/Adyen/adyen-magento2/pull/447/files#diff-aba9447e7c2f9df9662124e47fd30e2aR275
These 2 only, but it should be already in the codebase.